data/tlg0018/tlg001/tlg0018.tlg001.1st1K-eng1.xml

```diff
@@ -0,0 +1,1013 @@
+<div type="textpart" subtype="section" n="1">
+<p>
+
+<milestone unit="chapter" n="1"/> Of other lawgivers, some have set forth what they considered to be just and reasonable, in a naked and unadorned manner, while others, investing their ideas with an abundance of amplification, have sought to bewilder the people, by burying the truth under a heap of fabulous inventions.</p>
+</div>
+
+<div type="textpart" subtype="section" n="2">
+<p>But Moses, rejecting both of these methods, the one as inconsiderate, careless, and unphilosophical, and the other as mendacious and full of trickery, made the beginning of his laws entirely beautiful, and in all respects admirable, neither at once declaring what ought to be done or the contrary, nor (since it was necessary to mould beforehand the dispositions of those who were to use his laws) inventing fables himself or adopting those which had been invented by others.
+</p>
+</div>
+
+<div type="textpart" subtype="section" n="3">
+<p>And his exordium, as I have already said, is most admirable; embracing the creation of the world, under the idea that the law corresponds to the world and the world to the law, and that a man who is obedient to the law, being, by so doing, a citizen of the world, arranges his actions with reference to the intention of nature, in harmony with which the whole universal world is regulated.</p>
+</div>
+
+<div type="textpart" subtype="section" n="4">
+<p>Accordingly no one, whether poet or historian, could ever give expression in an adequate manner to the beauty of his ideas respecting the creation of the world; for they surpass all the power of language, <choice><sic>and amaze.</sic><corr>and amaze</corr></choice> our hearing, being too great and venerable to be adapted to the senses of any created being.</p>
+</div>
+
+<div type="textpart" subtype="section" n="5">
+<p>That, however, is not a reason for our yielding to indolence on the subject, but rather from our affection for the Deity we ought to endeavour to exert ourselves even beyond our powers in describing them: not as having much, or indeed anything to say of
+<pb n="2"/>
+our own, but instead of much, just a little, such as it may be probable that human intellect may attain to, when wholly occupied with a love of and desire for wisdom.
+</p>
+</div>
+
+<div type="textpart" subtype="section" n="6">
+<p>For as the smallest seal receives imitations of <choice><sic>things or</sic><corr>things of</corr></choice> colossal magnitude when engraved upon it, so perchance in some instances the exceeding beauty of the description of the creation of the world as recorded in the Law, overshadowing with its brilliancy the souls of those who happen to meet with it, will be delivered to a more concise record after these facts have been first premised which it would be improper to pass over in silence.
+<milestone unit="chapter" n="2"/></p>
```

Comment:
I would not leave these milestones at the end of a section. They should always go with the next container.

Reply:
I ended up going through and fixing these file by file.